### ISSUE TITLE
It should be possible to determine version of NUnit using nunit console tool

### ISSUE THREAD

**Issue by NikolayPianikov** [nunit/nunit#978]
Currently nunit console started without arguments outputs NUnit version. But it does not look like a reliable way to obtain version as this default behavior can easily change in the future. It would be great to have dedicated argument for the version.

**Comment by CharliePoole** [comment]
So would a --version option displaying the standard two-line header work for you?

**Comment by oznetmaster** [comment]
I would think a --version command would be easier to process if it returned a single line?

**Comment by NikolayPianikov** [comment]
I think it could be like this 3.0.5784

**Comment by CharliePoole** [comment]
If it's not too much trouble, I'd prefer to follow the Gnu standard for --version...
   http://www.gnu.org/prep/standards/standards.html#g_t_002d_002dversion
That makes more sense if somebody just type's it in at the command-line and isn't hard to parse.

Currently, our first two lines meet that standard:

NUnit Console Runner 3.0.5674
Copyright (C) 2015 Charlie Poole

In Debug, we add (Debug) at the end of the first line, but I could move that in front of the version so it's always at the end.

**Comment by rprouse** [comment]
While we are doing this, we should pull the copyright from the assemblyinfo
instead of hard coding it here and in nunitlite.

**Comment by oznetmaster** [comment]
We also should better define the AssemblyVersion then just "3.0.*".

Perhaps 3.<major revision>,<alpha/beta/rc number>.*

Using 0xx for alpha, 1xx for beta, and 2xx for RC?

So the current build would be AssemblyVersion "3.0.200.*".

**Comment by CharliePoole** [comment]
@rprouse We already do that. There is a hard-coded string that is used if no attribute is found. It's a little silly, but we have been doing it that way for many years. We do the same thing in NUnit and NUnitLite, so I might pull it out to a common class somewhere.

@oznetmaster Of course, we can't change what's already out there. :-)

Additionally, I don't think we can change how we version assemblies for 3.0 because the number we used (like 200) would be lower than the numbers that have been generated automatically for the alphas, betas and first rc. We could change how we do it for 3.1 and higher versions though.

**Comment by CharliePoole** [comment]
I've already modified some of this code for #922, so I'll take this one to avoid conflicts.

@NikolayPianikov Let me know if you're OK with what I proposed above. I'll start working on that basis, with --version and -v giving the two line output. If you strongly prefer to have just the number, we could do that as well.

**Comment by NikolayPianikov** [comment]
I think I can parse suggested text. The main thing this format should be stable from version to version. Thank you

**Comment by CharliePoole** [comment]
Great! We'll go ahead after I fix the items that came out in the review.

I think you can count on the first or only match for a three-component version number in the first line always being what you want. You are probably even safe to take the first instance of '2' or '3' as your guide, but that's a little loose.
